docs/pages/repo/docs/core-concepts/caching/what-to-cache.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: What to Cache?
+description: The configuration of Turborepo `outputs`
+---
+
+import Callout from "../../../../../components/Callout";
+
+# What to Cache?
+
+By default, the console output (also known as `stdout`) from all tasks is _always_ captured by Turborepo. If your task does not emit any files (e.g. unit tests with Jest) you can omit `outputs`. Beyond that, you will generally need to specify which files on disk you need to capture (and restore!) in order to be able to run subsequent dependent tasks.
+
+## Configuring Cache Outputs
+
+You configure cache outputs in Turborepo by passing specifying an array of globs inside [`pipeline.<task>.outputs`](/repo/docs/reference/configuration#outputs) as seen in this example:
+
+```jsonc filename="/turbo.json"
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "outputs": ["dist/**"]
+    }
+  }
+}
+```
+
+This idea is simple but can grow in complexity depending upon the tools you are using in your pipeline.
+
+### Inclusions
+
+In the above task definition for `build` it should be noted that it applies to _all_ of the `build` tasks in the entire Turborepo. It's quite likely that each of those individual tasks have slight differences in where their outputs are located, especially as the repository grows.
```

Comment:
This is a good discussion, but I think is better located in the workspace configurations docs. The link to get there is 3ish paragraphs and one example in, and probably should be up front. Something like "<Some note that this applies to _all_ `build` tasks>, to handle different workspaces that produce task outputs in different locations, we recommend Workspace Configurations" then move on.

"How to specify outputs" and "how to customize individual package-task configurations" are different (but related) topics, and I think this page would benefit with focusing on the former with links to the latter. I'm on board with improving our documentation of "how to customize individual package-task configurations" too.

Reply:
We've had multiple issues recently that resolve to "I have a root `build` task, each of my workspaces had a different output folder, none of which I specify in `outputs`, why isn't it working?"

Conceptually:
- I want to try hoisting this up for a bit. We probably lucky if we get 5 minutes of docs-reading per-user. I want to try to get this above the line for that five minutes and see how that ends up trading off with user success rate.
- I'm generally a fan of repetitiveness in docs.